core/src/test/java/apoc/create/CreateTest.java

```diff
@@ -315,6 +318,51 @@ public void testClonePathShouldNotEditOriginalOne() {
                     assertFalse(end.hasProperty("brazorf"));
                 });
     }
+
+    @Test
+    public void testClonePathShouldNotDuplicateRelsWithMultipaths() {
+        //create path with single rels
+        db.executeTransactionally("""
+                          CREATE (n:Node{id:0}),
+                            (n2:Node{id:1}),
+                            (n3:Node{id:2}),
+                            (n4:Node{id:3}),
+                            (n5:Node{id:4}),
+                            (n6:Node{id:5}),
+                            (n7:Node{id:6}),
+                            (n)-[:PARENT]->(n2),
+                            (n)-[:PARENT]->(n3),
+                            (n)-[:PARENT]->(n5),
+                            (n)-[:PARENT]->(n6)-[:PARENT]->(n7),
+                            (n)-[:PARENT]->(n4)-[:PARENT]->(n7)-[:PARENT]->(n2)-[:PARENT]->(n3)"""
+        );
+        
+        // returns a list with all rels
+        testCall(db, """
+                MATCH p=(n:Node{id: 0})-[:PARENT*..4]->(v:Node)
+                WITH collect(p) as p
+                CALL apoc.create.clonePathsToVirtual(p)
+                YIELD path
+                WITH collect( relationships(path) ) as pathRels
+                RETURN apoc.coll.flatten(pathRels) as rels""", r -> {
+            final List<Relationship> paths = (List<Relationship>) r.get("rels");
```

Comment:
I think `paths` is a bit of a confusing name here, as it is a list of relationships. How about `rels` or `relationships `instead?